deploy/infra/modules/aci.bicep

```diff
@@ -0,0 +1,114 @@
+@description('Name for the container group')
+param name string
+
+@description('Location for all resources.')
+param location string = resourceGroup().location
+
+@description('Container image to deploy. Should be of the form repoName/imagename:tag for images stored in public Docker Hub, or a fully qualified URI for other registries. Images from private registries require additional registry credentials.')
+param image string = 'mcr.microsoft.com/oss/mirror/docker.io/library/postgres:12.9-bullseye'
+
+@description('Port to open on the container and the public IP address.')
+param port int = 80
+
+@description('The number of CPU cores to allocate to the container.')
+param cpuCores int = 1
+
+@description('The amount of memory to allocate to the container in gigabytes.')
+param memoryInGb int = 4
+
+param userManagedIdentityId string
+
+param userManagedIdentityPrincipalId string
+
+param assignRoleToUserManagedIdentity string = 'Owner'
```

Comment:
Can we go with 'Contributor' as a default?

Reply:
I will capture this as backlog item.